tests/unit/conditions/test_jwt_condition.py

```diff
@@ -0,0 +1,146 @@
+from datetime import datetime, timezone
+
+import jwt
+import pytest
+from marshmallow import validates
+
+from nucypher.policy.conditions.base import ExecutionCall
+from nucypher.policy.conditions.jwt import JWTCondition, JWTVerificationCall
+
+TEST_ECDSA_PRIVATE_KEY_RAW_B64 = (
+    "MHcCAQEEIHAhM7P6HG3LgkDvgvfDeaMA6uELj+jEKWsSeOpS/SfYoAoGCCqGSM49\n"
+    "AwEHoUQDQgAEXHVxB7s5SR7I9cWwry/JkECIRekaCwG3uOLCYbw5gVzn4dRmwMyY\n"
+    "UJFcQWuFSfECRK+uQOOXD0YSEucBq0p5tA=="
+)
+
+TEST_ECDSA_PRIVATE_KEY = (  # TODO: Workaround to bypass pre-commit hook that detects private keys in code
+    "-----BEGIN EC"
+    + " PRIVATE KEY"
+    + f"-----\n{TEST_ECDSA_PRIVATE_KEY_RAW_B64}\n-----END EC"
+    + " PRIVATE KEY-----"
+)
+
+TEST_ECDSA_PUBLIC_KEY = (
+    "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEXHVxB7s5SR7I9cWwry"
+    "/JkECIReka\nCwG3uOLCYbw5gVzn4dRmwMyYUJFcQWuFSfECRK+uQOOXD0YSEucBq0p5tA==\n-----END PUBLIC "
+    "KEY-----"
+)
+
+ISSUED_AT = datetime.now(tz=timezone.utc)
+
+TEST_JWT_TOKEN = jwt.encode(
```

Comment:
should we test multiple algorithms?

Reply:
Unsure. I mean, we can, but at the same time, I'm not sure to what extent we're just testing the underlying library.

Reply:
yeah that's very fair